Basics/Git.md

```diff
@@ -4,50 +4,130 @@
+![Status chec][image-3]
+* ***Require branches to be up to date before merging***
+![Branches][image-4]
 
-If you want to use Git with Gitflow commands and terms, you might find the [git-flow][2] tool useful.
+## 7. Mobile specific
 
-[1]:	https://www.atlassian.com/git/tutorials/comparing-workflows#gitflow-workflow
-[2]:	https://github.com/nvie/gitflow
+Above mentioned rules should be the minimum for all platforms. Few more are specific and applicable only for mobile platforms. If you cannot apply some of the mentioned rules to your project, please talk with another platform and try to find a viable solution for both of you.
+
+### Git Flow/Infinum Flow
+
+We’ll be using the Git Flow standard as a base for git branching with some modifications. More about Git Flow [here][6].
+
+#### Branches
+We’ll be using **master** as the main branch. Since we’ll be using tags and optionally **release** branches for deployment, **develop** branch becomes redundant.
+
+The main branches in this flow are:
+* **master**
+  * It is a main branch
+  * anything in the master branch is deployable
+  * is stable and it is always, always safe to deploy from it or create new branches off of it
+  * feature and hotfix branches are merged into it only after pull request review is done
+* **release**
+  * used for parallel development (e.g. multiple versions of the app when version 1.0.0 is developed on master, version 1.1.0 can be a next ***release/1.1.0/*** or for developing multiple features that should or shouldn’t be in the same version: ***release/separate-cool-feature***)
+  * same rules as for the master branch
+* **feature**
+  * used for developing new features / adding new functionality to the app
+  * branches off of master/release
+  * has prefix ***feature/***, e.g. feature/123-add-followers-button
+* **fix**
+  * used for fixing non-critical production bugs or bugs occurred in development phase
+  * has prefix ***fix/***, e.g. fix/123-wrong-cards-sorting
+* **refactor**
+  * used for codebase/project refactoring
+  * Has prefix ***refactor/***, e.g. refactor/123-alert-and-action-sheet
+* **hotfix**
+  * used for fixing critical bugs in production
+  * may be branched off from the corresponding tag and create PR to the ***master*** or to the latest ***release*** branch if one exists
+  * has prefix ***hotfix/***, e.g. hotfix/1337-recipe-details-crash
+  * Example usage: there is a bug on app version 1.2.0. In that case you checkout to a commit tagged with v1.2.0. You create release/1.2.1 branch from that tag, and you create hotfix/whatever from that branch where you implement the fix and create PR back to the release/1.2.1 branch
+ 
+#### Pull requests
+Each pull request must have an automatic check - running tests or just building the app if your project doesn’t have tests. Setup on GitHub is described in general rules of this handbook, while on how to set up Bitrise Checks on projects you have [Android][7] and [iOS][8] handbook chapters.
```

Comment:
As this is iOS handbook, I'm not sure if Android link is needed here?

Reply:
It doesn't heart, and might be easier to transfer to shared repo if we decide to do that 😅